lib/graphql/language/nodes.rb

```diff
@@ -17,9 +17,6 @@ def initialize(options={})
           if options.key?(:position_source)
             position_source = options.delete(:position_source)
             @line, @col = position_source.line_and_column
-          elsif options.key?(:line)
```

Comment:
I thought it might have snuck in there for `graphql-libgraphqlparser` but looks like not: 
https://github.com/rmosolgo/graphql-libgraphqlparser-ruby/blob/master/ext/graphql_libgraphqlparser_ext/visitor_functions.c#L58-L63